mypaint-brush.c

```diff
@@ -22,6 +22,18 @@
 #include <math.h>
 #include <assert.h>
 
+#define GETTEXT_PACKAGE "libmypaint"
+
+#ifdef HAVE_GETTEXT
```

Comment:
I also think that this --without-glib really complexifies the build and wanted at some point propose to get rid of this (but then I thought "oh well, they must have their reasons" and abandoned). But yeah if there could be just a simple non-optional dependency, it would make code a lot clearer.